lib/msf/core/exploit.rb

```diff
@@ -273,6 +273,10 @@ def self.mixins
     return mixins
   end
 
+  class << self
```

Comment:
As far as I can tell, this isn't being used by anything since everything else is instance variables.

Reply:
I believe this is used below by the module cache:
```
@needs_cleanup      = module_instance.respond_to?(:needs_cleanup) && module_instance.needs_cleanup
```